bucket/mvndaemon.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.6.0",
+    "description": "The Maven (mvn) Daemon is based on GraalVM and allows for faster Maven builds. Needs a properly configured JAVA_HOME environment variable.",
```

Comment:
I would suggest you to move `Needs a properly configured JAVA_HOME environment variable.` into **notes**